eidas_node/tests/test_storage.py

```diff
@@ -9,19 +9,29 @@
 from eidas_node.utils import parse_xml
 
 
-class TestIgniteStorage(SimpleTestCase):
-    HOST = 'localhost.example.net'
-    PORT = 12345
-    REQUEST_CACHE_NAME = 'RequestCacheTest'
-    RESPONSE_CACHE_NAME = 'ResponseCacheTest'
+class IgniteMockMixin:
+    cache_mock = None  # type:  MagicMock
```

Comment:
`Mock` should be sufficient, but this looks like a change from another PR.

Reply:
This mixin was created to be shared with view tests, but I can move it to light storage PR.

Reply:
Yeah, I saw it later but haven't updated the comment. It can probably stay.